packages/jest-transform/src/ScriptTransformer.ts

```diff
@@ -43,10 +43,7 @@ const {version: VERSION} = require('../package.json');
 // This data structure is used to avoid recalculating some data every time that
 // we need to transform a file. Since ScriptTransformer is instantiated for each
 // file we need to keep this object in the local scope of this module.
-const projectCaches: WeakMap<
-  Config.ProjectConfig,
-  ProjectCache
-> = new WeakMap();
+const projectCaches = new Map<string, ProjectCache>();
```

Comment:
_probably_ not the correct fix, but with this change the tests pass at least, which is a good start 🙂

Reply:
Yeah I think this is alright.